index.js

```diff
@@ -424,6 +435,12 @@ alexa.app = function(name) {
           if (typeof eventHandlerObject != "undefined" && typeof eventHandlerObject["function"] == "function") {
             return Promise.resolve(eventHandlerObject["function"](request, response));
           }
+        } else if (request.isPlaybackController()) {
+          var playbackControllerEvent = requestType.slice(19);
```

Comment:
This is a total nitpick, but maybe these should be called `event` and `eventHandlerObject` just like above?